components/dashboard/src/teams/TeamBilling.tsx

```diff
@@ -188,14 +189,22 @@ export default function TeamBilling() {
                     <>
                         {availableTeamPlans.map((tp) => (
                             <>
-                                <SolidCard
-                                    className="cursor-pointer hover:bg-gray-200 dark:hover:bg-gray-700"
-                                    onClick={() => checkout(tp)}
-                                >
+                                <SolidCard className="w-72 h-72">
                                     <div className="px-2 py-5 flex-grow flex flex-col">
-                                        <div className="font-medium text-base">{tp.name}</div>
-                                        <div className="font-semibold text-gray-500 text-sm">Unlimited hours</div>
-                                        <div className="mt-8 font-semibold text-sm">Includes:</div>
+                                        <div className="font-semibold text-gray-800 dark:text-gray-100 text-lg">
+                                            {tp.name}
+                                        </div>
+                                        <div className="font-semibold text-gray-400 dark:text-gray-600 text-sm">
+                                            Unlimited hours
+                                        </div>
+                                        <div className="mt-2">
+                                            <PillLabel type="warn" className="font-semibold normal-case text-sm">
```

Comment:
**praise**: Don't know what you changed here but thanks for fixing the pill element here to hug contents instead of filling the container width.

Reply:
I removed a hard-coded `ml-2` from the component. 😇